bahmnicore-omod/src/main/java/org/bahmni/module/bahmnicore/web/v1_0/controller/BahmniConceptSearchController.java

```diff
@@ -0,0 +1,136 @@
+package org.bahmni.module.bahmnicore.web.v1_0.controller;
+
+import org.apache.commons.lang3.LocaleUtils;
+import org.bahmni.module.bahmnicore.service.BahmniDiagnosisService;
+import org.bahmni.module.terminologyservices.api.TerminologyLookupService;
+import org.openmrs.Concept;
+import org.openmrs.ConceptMap;
+import org.openmrs.ConceptName;
+import org.openmrs.ConceptReferenceTerm;
+import org.openmrs.ConceptSearchResult;
+import org.openmrs.ConceptSource;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.emrapi.concept.EmrConceptService;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.BaseRestController;
+import org.openmrs.util.LocaleUtility;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Locale;
+import java.util.Set;
+
+import static org.springframework.web.bind.annotation.ValueConstants.DEFAULT_NONE;
+
+@Controller
+@RequestMapping(value = "/rest/" + RestConstants.VERSION_1 + "/bahmniapi/")
```

Comment:
not sure about this url .. bahmniapi can be anything! we also needn't have any api (it is understood). For example
/bahmni/terminologies/
/bahmni/concepts/

Reply:
sure @angshu .. end point changed to **/bahmni/terminologies/concepts?term=asthma**